cmd/minikube/cmd/root.go

```diff
@@ -352,3 +353,10 @@ func applyToAllCommands(cmd *cobra.Command, f func(subCmd *cobra.Command)) {
 		}
 	}
 }
+
+// commandOptions creates minikube runtime options from the command line flags.
+// Flags that must be handled outside of the cmd package must be added to
+// run.Options.
+func commandOptions() *run.Options {
```

Comment:
This is moved to cmd/flags.Options in later commit since this function does not work for sharing options in cmd/config.